dags/hivemind_etl_helpers/github_summary_etl.py

```
@@ -0,0 +1,171 @@
+        for date in aggreagted_comment_docs.keys()
+    ]
+
+    prs_summarized: list[Document] = [
+        summarizer.transform_summary(
+            date=date,
+            summary=summarizer.process_commits(
+                date=date, documents=aggreagted_pr_docs[date]
+            ),
+            type=SummaryType.PR,
+        )
+        for date in aggreagted_pr_docs.keys()
```

Ruff (0.8.2)

Use `key in dict` instead of `key in dict.keys()`

Remove `.keys()`

(SIM118)